src/helpers/AuthHelper.ts

```diff
@@ -1 +1,36 @@
-export {};
+import config from '../config/environment'
+import { User } from '../interfaces/UserInterface'
+import jwt from 'jsonwebtoken'
+
+export class TokenGen {
+  payload: User
+  constructor(payload: User) {
+    this.payload = payload
+  }
```

Comment:
no need for the payload or the constructor here. also this be called AuthHelper!